prow/jobs/incubator/vstudio-extension/vstudio-extension.yaml

```diff
@@ -0,0 +1,45 @@
+job_template: &job_template
+  skip_report: true
+  decorate: true
+  path_alias: github.com/kyma-incubator/vstudio-extension
+  max_concurrency: 10
+  extra_refs:
+    - org: kyma-project
+      repo: test-infra
+      base_ref: master
+      path_alias: github.com/kyma-project/test-infra
+  spec:
+    containers:
+      - image: eu.gcr.io/kyma-project/prow/test-infra/buildpack-node:v20181130-b28250b
+        securityContext:
+          privileged: true
+        command:
+          - "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/build.sh"
+        args:
+          - "/home/prow/go/src/github.com/kyma-incubator/vstudio-extension"
+
+job_labels_template: &job_labels_template
+  preset-dind-enabled: "true"
+  preset-sa-gcr-push: "true"
+  preset-docker-push-repository-incubator: "true"
+
+presubmits: # runs on PRs
+  kyma-incubator/vstudio-extension:
+    - name: vstudio-extension
+      always_run: true
```

Comment:
do we need to specify always_run always when run_if_changed is not specifed? If so, should we review other jobs if always_run is set?

Reply:
Yes, it seems so: https://github.com/kubernetes/test-infra/blob/fd04fb95d09df002e87df9f137b199ac4b6cac77/prow/jobs.md#how-to-configure-new-jobs